src/components/login/LoginForm.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from "react";
+import { Button, FormGroup, FormControl, ControlLabel } from "react-bootstrap";
+import "./LoginForm.css";
+
+export default class LoginForm extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            email: "",
+            password: ""
+        };
+    }
+
+    validateForm() {
+        return this.state.email.length > 0 && this.state.password.length > 0;
+    }
+
+    handleChange = event => {
+        this.setState({
+            [event.target.id]: event.target.value
+        });
+    }
+
+    handleSubmit = event => {
```

Comment:
please add parentheses to `(event)`  for consistency. Because when you need more than one argument we can't avoid parentheses

Reply:
fixed